nntrainer/src/neuralnet.cpp

```diff
@@ -121,7 +120,6 @@ unsigned int parseType(std::string ll, input_type t) {
    *            "tanh"  : tanh
    *            "sigmoid" : sigmoid
    *            "relu" : relu
-   *            "softmax" : softmax
```

Comment:
Is there any reason removing softmax from the comment? `activation_string` still has softmax.

Reply:
Yes. fixed. thanks.